patches/chrome-browser-resources-extensions-detail_view.html.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/resources/extensions/detail_view.html b/chrome/browser/resources/extensions/detail_view.html
+index fc42d3289b47800da6e3884d6382c949e80f3ead..30aaf0cc92b68a2929ae20c84bed7d7ebf549854 100644
+--- a/chrome/browser/resources/extensions/detail_view.html
++++ b/chrome/browser/resources/extensions/detail_view.html
+@@ -351,7 +351,7 @@
+               <div class="section-content">$i18n{incognitoInfoWarning}</div>
+             </div>
+           </extensions-toggle-row>
+-        </template>
++        <extensions-brave-tor-toggle-row data="[[data]]" delegate="[[delegate]]">
```

Comment:
As suggested in DM, I suggest injecting the HTML and JS for the new toggle directly to the chromium component via `RegisterPolymerTemplateModifications` and `RegisterPolymerComponentBehaviors`, so that you can keep the directly-applied styles for the row instead of having to copy/paste. You also wouldn't need this patch or the .js file patch.

Reply:
fixed